Topics/System_Design/Mobile_App_System_Design.md

```diff
@@ -0,0 +1,33 @@
+# Essential Elements of Mobile App System Design
+
+## Introduction
+For the last 15 years, mobile apps have emerged as vital tools in our daily lives. The best mobile apps simplify our lives by helping us with daily tasks. What lies behind these apps is a thoughtful system design that has important considerations that differ from traditional desktop devices and must be taken into account when mobile app system design, namely: UI and Performance.
+
+
+## User Interface Design
+
+### 1. Screen Size
```

Comment:
With regards to this, you could also mention how designers have to keep in mind the various screen sizes of mobile devices such that they have to design their system (both technically and visually) in a way that is flexible and adaptable.

Reply:
That's a valid point. It really is an important issue! Thanks.